### PR TITLE
test: add some es-modules concatenation cases

### LINKED ISSUE #357: [Optimization request] Remove empty "modules"
When using this plugin we might end up with something like this:
<details>
<summary>Click</summary>
/***/ 54:
/***/ (function(module, exports, __webpack_require__) {

// extracted by mini-css-extract-plugin

/***/ }),

/***/ 92:
/***/ (function(module, exports, __webpack_require__) {

// extracted by mini-css-extract-plugin

/***/ }),

/***/ 93:
/***/ (function(module, exports, __webpack_require__) {

// extracted by mini-css-extract-plugin

/***/ }),

/***/ 94:
/***/ (function(module, exports, __webpack_require__) {

// extracted by mini-css-extract-plugin

/***/ }),

/***/ 95:
/***/ (function(module, exports, __webpack_require__) {

// extracted by mini-css-extract-plugin

/***/ }),

/***/ 96:
/***/ (function(module, exports, __webpack_require__) {

// extracted by mini-css-extract-plugin

/***/ }),

/***/ 97:
/***/ (function(module, exports, __webpack_require__) {

// extracted by mini-css-extract-plugin

/***/ }),

/***/ 98:
/***/ (function(module, exports, __webpack_require__) {

// extracted by mini-css-extract-plugin

/***/ }),
</details>

Seems like this could be optimized away - i know that probably minifier + gzip makes this kinda cheap, but parsing costs add up and it would be always good to strive for less code in such cases.

### PR DESCRIPTION
**Pull request by irudoy**
closes #357

This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [x] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### CODE REVIEW

File: test/cases/es-module-concatenation/expected/main.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+// extracted by mini-css-extract-plugin
+
+// CONCATENATED MODULE: ./index.js
+
+
+
+
+
+/***/ })
+/******/ ]);
```
Comment (evilebottnawi) [inline test/cases/es-module-concatenation/expected/main.js:111]: Just empty chunk?

Comment (irudoy) [reply test/cases/es-module-concatenation/expected/main.js:111]: yep, one instead of three

Comment (evilebottnawi) [reply test/cases/es-module-concatenation/expected/main.js:111]: Can you fix tests for webpack5? Just create webpack-4 and webpack-5 directors in expected, install webpack@5 and put expected result to webpack@5 :smile:

Anyway I found emitting empty chunk is bug and should be fixed for webpack@5, but it is not part this PR :+1:

Comment (vankop) [reply test/cases/es-module-concatenation/expected/main.js:111]: should be solved with webpack@5

**Pull request merged**